README.md

```diff
@@ -51,6 +51,7 @@
   - [Copy Pages](#copy-pages)
```

Comment:
* Would you please rename the `assets/pdfs/examples/file_with_attachment.pdf` file to `assets/pdfs/examples/add_attachments.pdf` to keep the naming convention consistent?
* Please remove the `package-lock.json` file from the PR. It's important to only have one dependency lock file. And since we use `yarn` for package management, that means we should only have the `yarn.lock` file.

> Neither of these things pertain to the `README` - but GitHub wouldn't let me comment on the files they apply to, so ¯\\_(ツ)_/¯

Reply:
No problem, sorry for the npm intrusion. Will keep to yarn with pdf-lib.